src/main/CommandLine.cpp

```diff
@@ -1024,6 +1050,9 @@ handleCommandLine(int argc, char* const* argv)
           runSignTransaction},
          {"upgrade-db", "upgade database schema to current version",
           runUpgradeDB},
+         {"replay-history-for-metadata",
```

Comment:
this should be an option for the `catchup` sub-command instead of being exposed at the top level like this

Reply:
Heh. The comment I put in for modes specifically says it's for when cases where it does something meaningfully different than an existing subcommand. I think in this case it really does! `catchup` modifies local state; `replay-history-for-metadata` is stateless, idempotent. From the user's perspective it's quite different than catchup; it just happens to do catchup internally. No?

Reply:
fixed, made into a `catchup` argument `--replay-in-memory`.